### PR TITLE
feat: toast notification #171

### PR DESCRIPTION
**Pull request by FabioSM46**
replace alerts with toast notifications using react-toastify

### CODE REVIEW

File: src/components/Blog/Blog.jsx [hunk 1]
```diff
@@ -44,14 +45,32 @@ const Blog = () => {
           return blog;
         });
       });
-      alert("You have liked the blog");
+      toast("You have liked the blog", {
```
Comment (hereisadi) [inline src/components/Blog/Blog.jsx]: toast.success would be more appropriate here

File: src/components/Blog/Blog.jsx [hunk 2]
```diff
@@ -44,14 +45,32 @@ const Blog = () => {
...
+        theme: "light",
+      });
     } catch (error) {
       if (
         error.response &&
         error.response.status === 400 &&
         error.response.data.error === "You have already liked this blog"
       ) {
-        alert("You have already liked this blog");
+        toast("You have already liked this blog", {
```
Comment (hereisadi) [inline src/components/Blog/Blog.jsx]: better to use toast.info here

File: src/components/Blog/Blog.jsx
Comment (hereisadi) [inline src/components/Blog/Blog.jsx:1]: Please change position to top-right and theme to dark for every toast message. Thanks.

**Review by hereisadi — CHANGES_REQUESTED** [review]

File: src/components/Blog/Blog.jsx (hunk 1 above)
Comment (FabioSM46) [reply src/components/Blog/Blog.jsx]: done

File: src/components/Blog/Blog.jsx (hunk 2 above)
Comment (FabioSM46) [reply src/components/Blog/Blog.jsx]: done

File: src/components/Blog/Blog.jsx
Comment (FabioSM46) [reply src/components/Blog/Blog.jsx:1]: done

**Comment by FabioSM46** [comment]
@hereisadi If you agree with the changes we can resolve the conversations. Thank you.

**Review by hereisadi — APPROVED** [review]

**Comment by hereisadi** [comment]
> @hereisadi If you agree with the changes we can resolve the conversations. Thank you.

Thanks, merging!

**Pull request merged**